applications/FluidDynamicsApplication/custom_strategies/strategies/compressible_navier_stokes_explicit_solving_strategy.h

```diff
@@ -242,7 +242,7 @@ class CompressibleNavierStokesExplicitSolvingStrategy
     {
         BaseType::InitializeSolutionStep();
 
-        if (ConservativeMagnitudeCalculationEnabled()) {
+        if (mFirstStep) {
```

Comment:
I'd put a comment in here saying that we do this in order to initialize the non-conservative values for the shock capturing calculation.